english/check-phone/open-message/open-message.md

```diff
@@ -3,3 +3,10 @@ white. Two black dots appear, slowly morphing into eyes as the white of
 the screen takes over your whole field of vision. 
 
 The marshmallows are here.
+What do you do?
+
+[Become a marshmallow.](english/becomemarshmallow/becomemarshmallow.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.